.gitignore

```diff
@@ -51,6 +51,7 @@ config/database.yml
 
 vcr.log
 .byebug_history
+config/app_environment_variables.rb
```

Comment:
@fdel15 nice idea, but I suggest to use environment variables, for this project we have figaro gem installed ([link](https://github.com/laserlemon/figaro)) this gem allows to define environments variables for all available environments in the app (like "development", "test" and "production")
So in this case, please take a moment to check the documentation of the gem and afterwards try to add application.yml to /config, the file is already in .gitignore by default: https://github.com/AgileVentures/LocalSupport/blob/develop/.gitignore#L54